[2024-06-28T22:43:17.211+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T22:43:17.230+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:43:13.186132+00:00 [queued]>
[2024-06-28T22:43:17.240+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:43:13.186132+00:00 [queued]>
[2024-06-28T22:43:17.241+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-28T22:43:17.252+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_delivery> on 2024-06-28 22:43:13.186132+00:00
[2024-06-28T22:43:17.259+0000] {standard_task_runner.py:63} INFO - Started process 466 to run task
[2024-06-28T22:43:17.261+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'data_delivery', 'manual__2024-06-28T22:43:13.186132+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpbembp5p2']
[2024-06-28T22:43:17.264+0000] {standard_task_runner.py:91} INFO - Job 164: Subtask data_delivery
[2024-06-28T22:43:17.304+0000] {task_command.py:426} INFO - Running <TaskInstance: NYC_Taxi_pipeline.data_delivery manual__2024-06-28T22:43:13.186132+00:00 [running]> on host 79a59ce16f65
[2024-06-28T22:43:17.542+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='data_delivery' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T22:43:13.186132+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T22:43:13.186132+00:00'
[2024-06-28T22:43:17.543+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T22:43:17.554+0000] {NYC_Taxi_pipeline.py:54} INFO - data_delivery started
[2024-06-28T22:43:17.554+0000] {Load_Data.py:20} INFO - Loading location Dimension ...
[2024-06-28T22:43:17.561+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.564+0000] {Load_Data.py:36} INFO - Existing records columns: ['locationid', 'borough', 'zone', 'service_zone', 'start_date', 'end_date', 'active_flag', 'version']
[2024-06-28T22:43:17.695+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.696+0000] {Table_creation.py:51} INFO - Loading data into table DimLocation...
[2024-06-28T22:43:17.699+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.701+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:43:17.701+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:43:17.702+0000] {Table_creation.py:64} INFO - insert new row to dimlocation
[2024-06-28T22:43:17.705+0000] {Table_creation.py:66} INFO - Data loaded successfully into table DimLocation With 3 Row
[2024-06-28T22:43:17.705+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.705+0000] {Load_Data.py:87} INFO - DimLocation dimension loaded successfully
[2024-06-28T22:43:17.705+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.706+0000] {Load_Data.py:95} INFO - loading DimRate Dimension ...
[2024-06-28T22:43:17.707+0000] {Table_creation.py:30} INFO - Loading data into table DimRate...
[2024-06-28T22:43:17.710+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.716+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimRate With 6 Row
[2024-06-28T22:43:17.717+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.717+0000] {Load_Data.py:98} INFO - DimRate_df dimension loaded successfully
[2024-06-28T22:43:17.717+0000] {Load_Data.py:113} INFO - loading payment dimension....
[2024-06-28T22:43:17.719+0000] {Table_creation.py:30} INFO - Loading data into table DimPayment...
[2024-06-28T22:43:17.722+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.727+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimPayment With 6 Row
[2024-06-28T22:43:17.727+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.728+0000] {Load_Data.py:116} INFO - payment_df dimension loaded successfully
[2024-06-28T22:43:17.728+0000] {Load_Data.py:11} INFO - loading Dimvendor Dimension ...
[2024-06-28T22:43:17.729+0000] {Table_creation.py:30} INFO - Loading data into table DimVendor...
[2024-06-28T22:43:17.733+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.738+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimVendor With 2 Row
[2024-06-28T22:43:17.738+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.738+0000] {Load_Data.py:14} INFO - Dimvedor_df dimension loaded successfully
[2024-06-28T22:43:17.738+0000] {Load_Data.py:104} INFO - loading trip_type dimension....
[2024-06-28T22:43:17.740+0000] {Table_creation.py:30} INFO - Loading data into table DimTrip_type...
[2024-06-28T22:43:17.743+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:43:17.747+0000] {Table_creation.py:45} INFO - Data loaded successfully into table DimTrip_type With 2 Row
[2024-06-28T22:43:17.747+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:43:17.747+0000] {Load_Data.py:107} INFO - trip_df dimension loaded successfully
[2024-06-28T22:43:17.747+0000] {Load_Data.py:122} INFO - loading fact table ....
[2024-06-28T22:43:21.262+0000] {Table_creation.py:30} INFO - Loading data into table Fact_Trip...
[2024-06-28T22:43:21.265+0000] {db_utils.py:21} INFO - Connected to PostgreSQL database successfully
[2024-06-28T22:46:24.839+0000] {Table_creation.py:45} INFO - Data loaded successfully into table Fact_Trip With 618792 Row
[2024-06-28T22:46:24.840+0000] {db_utils.py:30} INFO - PostgreSQL connection is closed
[2024-06-28T22:46:29.897+0000] {Load_Data.py:144} INFO - Fact_df  loaded successfully
[2024-06-28T22:46:29.930+0000] {NYC_Taxi_pipeline.py:61} INFO - data_delivery done
[2024-06-28T22:46:29.930+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-28T22:46:29.931+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T22:46:29.940+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_pipeline, task_id=data_delivery, run_id=manual__2024-06-28T22:43:13.186132+00:00, execution_date=20240628T224313, start_date=20240628T224317, end_date=20240628T224629
[2024-06-28T22:46:29.997+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T22:46:30.014+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T22:46:30.016+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
